tests/translator/output/aws-us-gov/function_with_alias_and_code_sha256.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
why did this file got changed? I am not able to understand how it is related to this change.